Mathlib/Algebra/Field/Shrink.lean

```diff
@@ -0,0 +1,26 @@
+/-
+Copyright (c) 2021 Kim Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kim Morrison
+-/
+import Mathlib.Algebra.Equiv.TransferInstance
+import Mathlib.Logic.Small.Defs
+import Mathlib.Tactic.SuppressCompilation
+
+/-!
+# Transfer field structures from `α` to `Shrink α`
+-/
+
+suppress_compilation
```

Comment:
I don't think we should use this in any new code.

Reply:
What alternative do you offer? `noncomputable section` is incompatible with `to_additive`

Reply:
`noncomputable` on each declaration, like we had before?

Reply:
Hmm, that's sad